darwin/importer/importer.py

```diff
@@ -821,11 +815,7 @@ def import_annotations(  # noqa: C901
     else:
         remote_classes = build_main_annotations_lookup_table(team_classes)
 
-    if dataset.version == 1:
-        console.print(
-            "Importing annotations...\nEmpty annotations will be skipped.", style="info"
-        )
-    elif dataset.version == 2 and delete_for_empty:
+    if dataset.version == 2 and delete_for_empty:
```

Comment:
```suggestion
    if delete_for_empty:
```

if `dataset.version` will never be 1 then we can the version check all together?

Reply:
Good spot!